src/kontrol/foundry.py

```diff
@@ -1382,3 +1390,14 @@ def init_project(project_root: Path, *, skip_forge: bool) -> None:
         logger=_LOGGER,
         cwd=root,
     )
+
+
+def foundry_clean(foundry: Foundry, options: CleanOptions) -> None:
+    if options.proofs and options.old_proofs:
+        raise AttributeError('Use --proofs or --old-proofs, but not both!')
```

Comment:
Does `options.proof` subsume `options.old_proofs`? If yes, maybe we can just use it if both are provided?

Reply:
I don't think it subsumes it, since  `options.proof` means "remove `proofs` unconditionally" and `options.old_proofs` means "remove `proofs` conditionally". But maybe there's another way of reading this?

Reply:
Ok, that’s fair, let’s leave it like this!